spec/features/registering_user_spec.rb

```diff
@@ -1,21 +1,28 @@
 require 'rails_helper'
 
 RSpec.describe "Registering user", type: :feature do
-  it 'shows a form' do
-    name = "funbucket13"
-    email = "funbucket13@gmail.com"
-    visit '/register'
 
-    click_on "Sign Up to Be a User"
-
-    expect(current_path).to eq(new_user_path)
+  before :each do
```

Comment:
For just registering a user, no user needs to be created. For checking logging in, yes.